docs/Installation-Guide.md

```diff
@@ -451,6 +468,10 @@ If your service uses a non-standard (i.e. not 80 or-443) port, make an internal
 
 ### Backup and restore
 
+Backups are important as you will eventually lose your data due many a thing. It is better to have them rather than be sorry. Yes, even for a simple service like tt-rss.
+
+Additionally, you should download your data in `OPML` regularity and restore from backups from time to time — is quarterly too much?
```

Comment:
It's good to mention OPML exporting as a suggestion for an additional, but very incomplete, form of backup, but the main focus should be on DB backups, and directly telling people they should regularly take and restore from OPML exports is too much.

Perhaps an additional section for OPML after `#### Restoring backups` would work.